Formula/tendermint.rb

```diff
@@ -0,0 +1,23 @@
+class Tendermint < Formula
+  desc "BFT state machine replication for applications in any programming languages"
+  homepage "https://tendermint.com/"
```

Comment:
```suggestion
  homepage "https://github.com/tendermint/tendermint"
```

tendermint.com is a company site...

Reply:
I think it is fine, we have some formulae pointing to company site for homepage.